upload-speed-100M.yaml

```diff
@@ -0,0 +1,6 @@
+uploadSpeed:
```

Comment:
you have to call this `smoke`, not `uploadSpeed`, same as below.

Reply:
OK, it's just that I could not find any documentation about what these files actually are and what the parameter names like `smoke` actually mean

Reply:
Also take note that I do not have write access to the `ethersphere/release-staging` repo. Maybe it should not be that restrictive? Should the whole team have access?

Reply:
This has been fixed yesterday.